### PR TITLE
Siemens CVE update for AD-2021-02

### PR DESCRIPTION
**Pull request by productcert**

### CODE REVIEW

**Comment by ATweedMITRE** [comment]
Thanks, these records have been updated.

File: 2020/26xxx/CVE-2020-26989.json [hunk 1]
```diff
@@ -25,17 +25,17 @@
                                 }
```
Comment (cve-team) [inline 2020/26xxx/CVE-2020-26989.json:25]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/26xxx/CVE-2020-26989.json:25]: ('Expecting property name enclosed in double quotes: line 103 column 13 (char 4261)',)

File: 2020/26xxx/CVE-2020-26990.json [hunk 2]
```diff
@@ -19,27 +19,7 @@
                                 "version": {
```
Comment (cve-team) [inline 2020/26xxx/CVE-2020-26990.json:19]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/26xxx/CVE-2020-26990.json:19]: ('Expecting property name enclosed in double quotes: line 78 column 13 (char 3025)',)

File: 2020/26xxx/CVE-2020-26991.json [hunk 3]
```diff
@@ -19,27 +19,7 @@
                                 "version": {
```
Comment (cve-team) [inline 2020/26xxx/CVE-2020-26991.json:19]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/26xxx/CVE-2020-26991.json:19]: ('Expecting property name enclosed in double quotes: line 78 column 13 (char 3015)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Pull request merged**